.clitest/functions/.test.makeplugin

```diff
@@ -0,0 +1,6 @@
+#!/bin/zsh
+
+local file=${1:-foo.plugin.zsh}
+local plugin=${2:-${1%%.*}}
+mkdir -p $T_TEMPDIR/$plugin
+echo "echo 'sourcing ${plugin}/${file}...'" >$T_TEMPDIR/$plugin/$file
```

Comment:
`print "print 'sourcing $plugin/$file...'"` would be more canonical in Zsh. Also, I prefer to have a space between `>` and a filename.

Reply:
I would appreciate if you could still replace all instances of `echo` with `print -r`. 🙂

Reply:
Sorry, missed that one.